Lottie-Windows/Lottie-Windows.csproj

```
@@ -9,21 +9,21 @@
     <Description>This library provides the LottieVisualSource which is consumed by the Microsoft.UI.Xaml.Controls.AnimatedVisualPlayer to render Lottie JSON files.</Description>
     <PackageTags>UWP Toolkit Windows Animations Lottie XAML</PackageTags>
 
-    <RootNamespace>Microsoft.Toolkit.Uwp.UI.Lottie</RootNamespace>
-    <AssemblyName>Microsoft.Toolkit.Uwp.UI.Lottie</AssemblyName>
+    <RootNamespace>CommunityToolkit.WinUI.Lottie</RootNamespace>
+    <AssemblyName>CommunityToolkit.WinUI.Lottie</AssemblyName>
```

```suggestion
    <AssemblyName>CommunityToolkit.WinUI.Lottie</AssemblyName>
    <PackageId>CommunityToolkit.Uwp.Lottie</PackageId>
```

I think it's fine to have the Assembly Name match the namespace still, but we'll want to set the PackageId as otherwise the package names will conflict with the new WinUI 3 one.